### PR TITLE
Add new cmdlet files

### PR DESCRIPTION
**Pull request by srinidhikMS**
Get-CsBatchTeamsDeployment
New-CsBatchTeamsDeployment

### CODE REVIEW

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+### EXAMPLE 1
+```
+New-CsBatchTeamsDeployment -TeamsFilePath "C:\dscale\Teams.csv" -UsersFilePath "C:\dscale\Users.csv" -UsersToNotify "adminteams@contoso.com,adelev@contoso.com"
+```
+
+## PARAMETERS
+
+### -TeamsFilePath
+The path to the teams.csv file.
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:37]:
```suggestion
The path to the CSV file that defines the teams you're creating.
```

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UsersFilePath
+The path to the users.csv file.
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:52]:
```suggestion
The path to the CSV file that maps the users you're adding to each team.
```

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UsersToNotify
+The tenant admin emailId to send the completion report of the batch deployment.
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:67]: According to https://docs.microsoft.com/en-us/microsoft-365/frontline/deploy-teams-at-scale you can specify "the email addresses of up to five recipients to notify about this deployment." 

does it have to be admins? 
can this description be expanded/improved?

**Review by get-itips — CHANGES_REQUESTED** [review]
Kindly see my comments thanks

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+## SYNTAX
+
+```
+Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 
+
+To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/en-us/microsoft-365/frontline/deploy-teams-at-scale?view=o365-worldwide).
```
Comment (dariomws) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:23]:
```suggestion
To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).

```

**Review by dariomws — CHANGES_REQUESTED** [review]
Will these cmdlets be available in the next GA version of the TPM?

**Comment by srinidhikMS** [comment]
> Will these cmdlets be available in the next GA version of the TPM?

Yes. This feature will be GA-ed on Aug 31st. Thanks.

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md (hunk 3 above)
Comment (srinidhikMS) [reply teams/teams-ps/teams/New-CsBatchTeamsDeployment.md]: Thank you for your review. You are right. This solution is mostly used by tenant admins but there will be no restriction for other users. I have updated the description.

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md (hunk 2 above)
Comment (srinidhikMS) [reply teams/teams-ps/teams/New-CsBatchTeamsDeployment.md]: Updated. Thanks.

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md (hunk 1 above)
Comment (srinidhikMS) [reply teams/teams-ps/teams/New-CsBatchTeamsDeployment.md]: Updated. Thanks.

**Review by dariomws — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## SYNTAX
+
+```
+Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 
+
+To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).
```
Comment (tseward) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:23]: I would use markdown for this, i.e.:

```
see [Deploy Teams at scale for frontline workers](https://docs.microsoft.com.......)].

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Get-CsBatchTeamsDeployment -OrchestrationId "My-Orchestration-Id"
+```
+
```
Comment (tseward) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:33]: Can you add a brief description of the example?

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+- Add users to teams as owners or members.
+- Manage teams at scale by adding or removing users from existing teams.
+- Stay notified through email, including completion, status, and errors (if any). You can choose to notify up to five people about the status of each batch of teams you deploy. Team owners and members are automatically notified when they're added to a team.
+## EXAMPLES
+
+### EXAMPLE 1
+```
+New-CsBatchTeamsDeployment -TeamsFilePath "C:\dscale\Teams.csv" -UsersFilePath "C:\dscale\Users.csv" -UsersToNotify "adminteams@contoso.com,adelev@contoso.com"
+```
+
```
Comment (tseward) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:33]: Can you add a brief description of the example?

**Review by tseward — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md (hunk 5 above)
Comment (srinidhikMS) [reply teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md]: Updated. Thanks.

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md (hunk 7 above)
Comment (srinidhikMS) [reply teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:34]: Done. Thanks.

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+```
+Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 
+
+To learn more, see [Deploy Teams at scale for frontline workers](https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).
+
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:25]:
```suggestion
```

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Get-CsBatchTeamsDeployment -OrchestrationId "My-Orchestration-Id"
+```
+This command provides the status of specified batch deployment orchestrationId.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:33]:
```suggestion
This command provides the status of the specified batch deployment orchestration ID.

```

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+## PARAMETERS
+
+### OrchestrationId
+This ID is generated when a batch deployment is submitted with New-CsBatchTeamsDeployment cmdlet.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:38]:
```suggestion
This ID is generated when a batch deployment is submitted with the New-CsBatchTeamsDeployment cmdlet.

```

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+## OUTPUTS
+###Status of the orchestrationId
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:53]:
```suggestion
### Status of the orchestration ID

```

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+Running: The orchestration is running.
+Completed: The orchestration is completed, either succeeded, partially succeeded or Failed.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:56]:
```suggestion
Completed: The orchestration is completed, either succeeded, partially succeeded, or failed.

```

File: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md:58]:
```suggestion
```

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+ -UsersToNotify <string> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deploying Teams at scale enables admins to deploy up to 500 teams and add 25 users per team using one Teams PowerShell command and two CSV files. This allows admins to meet your organization's scale needs and significantly reduce deployment time. Admins can also use this solution to add and remove members from existing teams at scale. You can use this cmdlet to:
+- Create teams using pre-built templates or your own custom templates.
+- Add users to teams as owners or members.
+- Manage teams at scale by adding or removing users from existing teams.
+- Stay notified through email, including completion, status, and errors (if any). You can choose to notify up to five people about the status of each batch of teams you deploy. Team owners and members are automatically notified when they're added to a team.
+## EXAMPLES
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:27]:
```suggestion

## EXAMPLES

```

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+
+### EXAMPLE 1
+```
+New-CsBatchTeamsDeployment -TeamsFilePath "C:\dscale\Teams.csv" -UsersFilePath "C:\dscale\Users.csv" -UsersToNotify "adminteams@contoso.com,adelev@contoso.com"
+```
+This command runs a batch deployment with the provided parameters in the CSV files and emails the status and errors(if any) to adminteams@contoso.com and adelev@contoso.com.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:33]:
```suggestion
This command runs a batch deployment with the provided parameters in the CSV files and emails the status and errors (if any) to adminteams@contoso.com and adelev@contoso.com.

```

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+Accept wildcard characters: False
+```
+
+## INPUTS
+
+## OUTPUTS
+### OrchestrationId
+The ID of the operation that can be used with the Get-CsBatchTeamsDeployment cmdlet to get the status of the operation.
+
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:88]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @srinidhikMS, kindly commit the suggested minor changes. Many thanks.

File: teams/teams-ps/teams/New-CsBatchTeamsDeployment.md (hunk 14 above)
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsBatchTeamsDeployment.md:27]:
```suggestion

## EXAMPLES

```

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@srinidhikMS Please implement the suggested copy review.Thanks

**Comment by srinidhikMS** [comment]
> @srinidhikMS Please implement the suggested copy review.Thanks

Fixed. Thanks.

**Comment by srinidhikMS** [comment]
> Copy review - @srinidhikMS, kindly commit the suggested minor changes. Many thanks.

Fixed now. Thanks for the review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@srinidhikMS 
Hi,

At your earliest convenience, please complete your account setup so that your GitHub and Microsoft accounts are connected:
- [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main)

Thanks!

**Pull request merged**